lab-matt/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-matt",
+  "version": "0.0.1",
+  "description": "Cowsay HTTP requests",
+  "main": "cowsay.js",
+  "scripts": {
+    "test": "lint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/muddybeard210/lab-06-cowsay-http-server.git"
+  },
+  "keywords": [
+    "cowsay"
+  ],
+  "author": "Matthew Sheridan-Smith",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/muddybeard210/lab-06-cowsay-http-server/issues"
+  },
+  "homepage": "https://github.com/muddybeard210/lab-06-cowsay-http-server#readme",
+  "devDependencies": {
```

Comment:
You need to add 'gulp' to your devDependencies and move 'cowsay' from you devDependencies into your dependencies since your server requires it to run.